win/build.bat

```diff
@@ -0,0 +1,278 @@
+if ERRORLEVEL 1 echo Cannot check build number. & GOTO :EOF
+for /f "tokens=1,2,3,4 delims=." %%a IN (%SOURCE_DIR%\%VERSION_FILE%) DO (
+  set MAJOR_VERSION=%%a
+  set MINOR_VERSION=%%b
+  set PATCH_VERSION=%%c
+  set EXTRA_VERSION=%%d
+)
+if NOT "%EXTRA_VERSION%." == "." (
+  for /f "tokens=1,* delims=-" %%a IN ("%EXTRA_VERSION%") DO set SERIAL_NUMBER=%%a
+) else (
+  if EXIST "%SOURCE_DIR%\.git" (
+    for /f "delims=" %%i in ('"%GIT_PATH%" rev-list --count --after %CCI_VERSION_START_DATE% HEAD %CCI_VERSION_SRC_LIST%') do set SERIAL_NUMBER=0000%%i
+    for /f "delims=" %%i in ('"%GIT_PATH%" rev-parse HEAD') do set HASH_TAG=%%i
+  ) else (
+    set EXTRA_VERSION=0000
+    set SERIAL_NUMBER=0000
+  )
+)
+set SERIAL_NUMBER=%SERIAL_NUMBER:~-4%
+
+if NOT "%HASH_TAG%." == "." set HASH_TAG=%HASH_TAG:~0,7%
+
+if NOT "%HASH_TAG%." == "." set EXTRA_VERSION=%SERIAL_NUMBER%-%HASH_TAG%
+
+echo Build Version is [%VERSION% (%MAJOR_VERSION%.%MINOR_VERSION%.%PATCH_VERSION%.%EXTRA_VERSION%)]
+set VERSION=%MAJOR_VERSION%.%MINOR_VERSION%.%PATCH_VERSION%.%EXTRA_VERSION%
+set BUILD_NUMBER=%MAJOR_VERSION%.%MINOR_VERSION%.%PATCH_VERSION%.%EXTRA_VERSION%
+
+if "%BUILD_TARGET%" == "Win32" (set CUBRID_CCI_PACKAGE_NAME=CUBRID-CCI-Windows-x86-%VERSION%) ELSE set CUBRID_CCI_PACKAGE_NAME=CUBRID-CCI-Windows-x64-%VERSION%
+
+set BUILD_ROOT_DIR=%SOURCE_DIR%\Build_Win
+set BUILD_DIR=%BUILD_ROOT_DIR%\build_%BUILD_MODE%_%BUILD_TARGET%_%BUILD_GEN_VERSION%
+if NOT EXIST "%BUILD_ROOT_DIR%" md %BUILD_ROOT_DIR%
+if NOT EXIST "%BUILD_DIR%" md %BUILD_DIR%
+
+rem TODO move build_prefix
+set BUILD_ROOT_PREFIX=%SOURCE_DIR%\win\output
+set BUILD_PREFIX=%BUILD_ROOT_PREFIX%\CUBRID_%BUILD_MODE%_%BUILD_TARGET%_%BUILD_GEN_VERSION%
+echo Build install directory is [%BUILD_PREFIX%].
+
+if "%DIST_DIR%." == "." set DIST_DIR=%BUILD_DIR%\output
+call :ABSPATH "%DIST_DIR%" DIST_DIR
+echo Packages Output directory is [%DIST_DIR%].
+if NOT EXIST "%DIST_DIR%" md %DIST_DIR%
+GOTO :EOF
+
+
+:BUILD_CUBRID
+echo Building CUBRID in %BUILD_DIR%
+cd /d %BUILD_DIR%
+
+rem TODO: get generator from command line
+if "%BUILD_TARGET%" == "Win32" (
+  set CMAKE_GENERATOR=%BUILD_GENERATOR%
+) else (
+  set CMAKE_GENERATOR="%BUILD_GENERATOR:"=% Win64"
+)
+echo CMAKE_GENERATOR is [%CMAKE_GENERATOR%]
+
+if "%BUILD_GEN_VERSION%" == "V141" (
+  "%CMAKE_PATH%" -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX=%BUILD_PREFIX% -DPARALLEL_JOBS=10 -DCUBRID_CCI_PACKAGE_NAME=%CUBRID_CCI_PACKAGE_NAME% %SOURCE_DIR%
+) else (
+  "%CMAKE_PATH%" -G %CMAKE_GENERATOR% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX=%BUILD_PREFIX% -DPARALLEL_JOBS=10 -DCUBRID_CCI_PACKAGE_NAME=%CUBRID_CCI_PACKAGE_NAME% -DFOR_OTHER_DRIVER=true %SOURCE_DIR%
+)
+
+if ERRORLEVEL 1 (echo FAILD. & GOTO :EOF) ELSE echo OK.
+
+"%CMAKE_PATH%" --build . --config %BUILD_TYPE% --target install
+if ERRORLEVEL 1 (echo FAILD. & GOTO :EOF) ELSE echo OK.
+
+GOTO :EOF
+
+
+:BUILD_CCI_PACKAGE
```

Comment:
I compiled CCI in Widnows and checked the result.
The cascci.dll file is created in this path: cubrid-cci\Build_Win\build_Release_x64_V141\bin\RelWithDebInfo
The cascci.lib file is created in this path: cubrid-cci\Build_Win\build_Release_x64_V141\cci\RelWithDebInfo.
Can't two files be created in the same path?

Reply:
This result is same to the engine build result.
[Cmake Build Source]
‘’’ cci/CMakeLists.txt
install(TARGETS cascci
  RUNTIME DESTINATION ${CUBRID_CCIDIR}/bin COMPONENT CCI
  LIBRARY DESTINATION ${CUBRID_CCIDIR}/lib COMPONENT CCI
  ARCHIVE DESTINATION ${CUBRID_CCIDIR}/lib COMPONENT CCI
  PUBLIC_HEADER DESTINATION ${CUBRID_CCIDIR}/include COMPONENT CCI
)
‘’’
Why do you want them(dll,lib) to be created in the same path?

For release, files is created in the following path and packaged.
Output : D:\work\fork\test\cubrid-cci\win\output\CUBRID_Release_x64_V141
Packaged : D:\work\fork\test\cubrid-cci\Build_Win\build_Release_x64_V141\output

Reply:
I thought it was different from the existing release method.

Reply:
ok, Thank for review